geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/tables/AttributeTable.scala

```diff
@@ -51,7 +51,7 @@ object AttributeTable extends GeoMesaTable with LazyLogging {
   override def supports(sft: SimpleFeatureType) =
     sft.getSchemaVersion > 5 && sft.getAttributeDescriptors.exists(_.isIndexed)
 
-  override val suffix: String = "attr_idx"
+  override val suffix: String = "attr"
```

Comment:
Does this affect 'existing' installations?  I'm guessing 'no' since the table name would be configured in the metadata in Accumulo.